### PR TITLE
Possible recommend screen

### PR DESCRIPTION
**Pull request by JoaquinPolonuer**
Recommend by id screen. Code quality should be improved

### CODE REVIEW

File: pages/recommend.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import Head from "next/head";
+import styles from "../styles/Recommend.module.css";
+import React from "react";
+import SearchBar from "../components/home/SearchBar";
+import BookCard from "../components/recommended/BookCard";
+import { useRouter, withRouter } from "next/router";
+
+export default withRouter(
```
Comment (JoaquinPolonuer) [inline pages/recommend.js:8]: Using withRouter as wrapper... Got it from https://stackoverflow.com/a/67637875, dont trust to much

File: components/home/SearchBar.js [hunk 2]
```diff
@@ -11,29 +11,50 @@ export default class SearchBar extends React.Component {
...
-      console.log((res) => console.log(res.json()))
-    );
+    fetch("http://localhost:8000/books/search_book/" + this.state.search)
+      .then((res) => res.json())
+      .then((res) => this.setState({ matches: res.data.slice(0, 5) }));
+  };
+
+  goToRecommend = (book_id) => {
+    //navigate to /recommend?book_id=book_id
+    window.location.href = `/recommend?book_id=${book_id}`;
```
Comment (JoaquinPolonuer) [inline components/home/SearchBar.js:21]: Maybe we could implement a better navigation/routing system

File: pages/recommend.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        recommended: [],
+      };
+    }
+
+    handleCheckBox = (e) => {
+      this.setState({ [e.target.name]: !this.state[e.target.name] });
+    };
+
+    getRecommended = () => {
+      const book_id = this.props.router.asPath.split("book_id=")[1];
```
Comment (JoaquinPolonuer) [inline pages/recommend.js:23]: Don't like it... Didn't find any other quick workaround and wanted to get it done

File: components/home/SearchBar.js (hunk 2 above)
Comment (LucaJaich) [reply components/home/SearchBar.js:21]: https://nextjs.org/docs/api-reference/next/link

File: pages/recommend.js (hunk 3 above)
Comment (LucaJaich) [reply pages/recommend.js:23]: https://coderrocketfuel.com/article/retrieve-query-string-parameters-from-a-url-in-next-js

**Pull request merged**